data/json/items/tool/misc.json

```
@@ -58,6 +83,70 @@
     "color": "brown",
     "use_action": "BELL"
   },
+  {
+    "type": "TOOL",
+    "id": "eac_dab_pen",
+    "name": "dab pen",
+    "name_plural": "dab pens",
+    "description": "A battery operated dab pen used for smoking cannabis distillate cartridges. ",
```

Ugh, I'm going to wind up learning about vape culture reviewing this, aren't I. ;-)
```suggestion
    "description": "A battery operated dab pen used for smoking cannabis distillate cartridges.",
```

---

Yep ur gonna have to ;-)